apps/telegram-bot/core/handlers/url_process.py

```
@@ -0,0 +1,225 @@
+                special_function_keyboard,
+                basic_function_keyboard,
+            ]
+            reply_markup = InlineKeyboardMarkup(keyboard)
+            await process_message.reply_text(
+                f"For the {i + 1}th url: {url}, please choose the function you want to use:",
+                reply_markup=reply_markup,
+            )
+            await process_message.delete()
+
+
+async def https_url_auto_process(update: Update, context: CallbackContext) -> None:
```

Ruff (0.15.1)

[warning] Unused function argument: `context`

(ARG001)